runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsConfig.java

```diff
@@ -28,20 +28,78 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.Range;
+import net.runelite.client.config.ConfigSection;
 
 @ConfigGroup("inventorytags")
 public interface InventoryTagsConfig extends Config
 {
-	enum DisplayMode
+	String GROUP = "inventorytags";
+
+	int OPACITY_LIMIT_MIN = 0;
+	int OPACITY_LIMIT_MAX = 255;
+
+	@ConfigSection(
+			name = "Tag style",
+			description = "The tag style of highlighted items",
+			position = 0
+	)
+	String tagStyleSection = "tagStyleSection";
+
+	@ConfigItem(
+			position = 0,
+			keyName = "showTagOutline",
+			name = "Outline",
+			description = "Configures whether or not item tags show be outlined",
+			section = tagStyleSection
+	)
+	default boolean showTagOutline()
 	{
-		OUTLINE,
-		UNDERLINE
+		return true;
 	}
 
-	String GROUP = "inventorytags";
+	@ConfigItem(
+			position = 1,
+			keyName = "tagFill",
+			name = "Fill",
+			description = "Configures whether or not item tags should be filled",
+			section = tagStyleSection
+	)
+	default boolean showTagFill()
+	{
+		return false;
+	}
+
+	@Range(
+			min = OPACITY_LIMIT_MIN,
+			max = OPACITY_LIMIT_MAX
+	)
+	@ConfigItem(
+			position = 2,
+			keyName = "alphaValue",
+			name = "Opacity Level",
+			description = "Configures the opacity of each color",
+			section = tagStyleSection
+	)
+	default int getAlphaValue()
+	{
+		return 50;
+	}
```

Comment:
Given the fill + alpha configs must be paired together, I think this order would be better for the modes:

1. Outline
2. Underline
3. Fill
4. Fill opacity